README.md

```diff
@@ -151,3 +151,21 @@ to create a 'Service' and a 'Profile' tied to the developer portal accounts.
 
 You should receive an API-Key (in the developer portal) and an email
 with a fake fiscal code you can use to start testing the Digital Citizenship API.
```

Comment:
question, are the fake fiscal codes actually valid fiscal codes? (i.e. do we have a non-zero probability of some real person having that fiscal code?)

Reply:
fake fiscal codes cannot collide with real ones as we use an "invalid" character ("Y") for the local code: https://github.com/gunzip/digital-citizenship-onboarding/blob/master/src/fake_profile.ts#L17